MechanicsCore/src/main/java/me/deecaad/core/mechanics/conditions/OnGroundCondition.java

```diff
@@ -0,0 +1,29 @@
+package me.vierdant.bridgedmechanics.tmep;
+
+import me.deecaad.core.file.SerializeData;
+import me.deecaad.core.file.SerializerException;
+import me.deecaad.core.mechanics.CastData;
+import me.deecaad.core.mechanics.conditions.Condition;
+import org.jetbrains.annotations.NotNull;
+
+public class OnGroundCondition extends Condition {
```

Comment:
`OnBlock` and `OnGround` need to be merged... less clutter on the wiki.

Imagine
```yaml
Mechanics:
  - "?OnGround{}"  # Checks if the entity is on the ground
  - "?OnGround{blocks=[DIRT, GRAVEL]}"  # Checks if the entity is on the ground AND that block is dirt or gravel. 
```